src/uu/factor/src/factor.rs

```diff
@@ -156,4 +156,12 @@ mod tests {
             .map(|i| 2 * i + 2u64.pow(32) + 1)
             .all(|i| factor(i).product() == i));
     }
+
+    #[test]
+    fn factor_recombines_strong_pseudoprime() {
```

Comment:
Maybe add a comment explaining the why ?

Reply:
Thanks for the suggestion, and done  :)